src/Files.App/Views/Shells/ModernShellPage.xaml

```diff
@@ -84,11 +100,13 @@
 				<VisualState x:Name="ShellBackgroundFocusOnState">
 					<VisualState.Setters>
 						<Setter Target="RootGrid.Background" Value="{ThemeResource App.Theme.FileArea.BackgroundBrush}" />
+						<Setter Target="RootGrid.BorderThickness" Value="{ThemeResource Local.ActivePane.BorderThickness}" />
```

Comment:
We need a way to only apply this resource when dual pane is enabled.

Reply:
It seems that we'll need another Visual State for when Dual Pane is turned off.

Reply:
Yeah this is for background so we can create ShellBorderThicknessFocusOnState and OffState.